### PR TITLE
Cancer Analysis based on Chernobyl Event 

### PR DESCRIPTION
**Pull request by swadhin1237**
# Pull Request for ML-Crate 💡

## Issue Title: Cancer Analysis based on Chernobyl Event

- **Info about the related issue (Aim of the project)** :  TO ANALYSIS THE CANCER RATE BEFORE AND AFTER CHERNOBYL EVENT
- **Name:** SWADHIN SATYAPRAKASH MAJHI
- **GitHub ID:** https://github.com/swadhin1237
- **Idenitfy yourself:** Contributor

Closes: #391 

###  1.Added the EDA and analysis file for the project with proper maintain the code of conduct of the repository
   📃 2.The dataset in the dataset folder
         3. The images in the image folder
         4. The requirement txt

Give a clear description of what have you added or modifications made

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ⚙️

I have checked from my side and it is based on the issue that has been assigned to me and it have no error.

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: Cancer_rate before and after Chernobyl/Model/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Cancer-rate-before-and-after-Chernobyl
```
Comment (abhisheks008) [inline Cancer_rate before and after Chernobyl/Model/README.md:1]: Project name should be, Cancer Analysis based on Chernobyl Event

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Everything looks good to me. Few more changes are required to get it perfectly done.
1. Rename the folder as, `Cancer Analysis based on Chernobyl Event`.
2. Add the visualization images inside the `README.md` file under the **Visualization** heading. You can use this syntax to put the images in the markdown file, `name of the image`.

**Comment by abhisheks008** [comment]
image

You have messed up with the project structure. The project structure might be as follows,
```
Cancer Analysis based on Chernobyl Event
|-- Model
   |-- cancer_analysis_based_on_Chernobyl_event.ipynb
   |-- README.md
|-- Images
   |-- visualization_image1.png
   |-- visualization_image2.png
   |-- visualization_image3.png
   |-- visualization_image4.png
|-- Dataset
   |-- dataset.csv
|-- requirements.txt
```

**Comment by swadhin1237** [comment]
I have done all 3 changes could you please verified and accept the pull request

**Review by abhisheks008 — APPROVED** [review]
Nicely done!
Your PR is approved and ready to be merged.
@swadhin1237

**Pull request merged**